simple_history/models.py

```diff
@@ -55,10 +67,13 @@ def __init__(
         history_id_field=None,
         history_change_reason_field=None,
         user_model=None,
-        get_user=default_get_user,
+        get_user=_default_get_user,
         cascade_delete_history=False,
         custom_model_name=None,
         app=None,
+        history_user_id_field=None,
+        history_user_getter=_history_user_getter,
+        history_user_setter=_history_user_setter,
```

Comment:
This `__init__` is getting pretty unwieldy. Can we coalesce these arguments better? Maybe a `FieldOverrides` kind of object to group those together.

Reply:
Decided to continue adding params for version 2.7. Will open an issue for figuring out a better way to handle and will deprecate before removing in 3